modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/configuration/admin/service/DLSizeLimitManagedServiceFactory.java

```diff
@@ -48,9 +39,7 @@
 @Component(
 	configurationPid = "com.liferay.document.library.internal.configuration.DLSizeLimitConfiguration",
 	property = Constants.SERVICE_PID + "=com.liferay.document.library.internal.configuration.DLSizeLimitConfiguration.scoped",
-	service = {
-		DLSizeLimitManagedServiceFactory.class, ManagedServiceFactory.class
-	}
+	service = ManagedServiceFactory.class
 )
 public class DLSizeLimitManagedServiceFactory implements ManagedServiceFactory {
 
```

Comment:
@RenanUmV @MrJohn1911, please rename to `DLSizeLimitHelper` and register the `ManagedServiceFactory` service dynamically into the activate method. Thx